### PR TITLE
update formatting of code blocks

### PR DESCRIPTION
**Pull request by kbhawkey**
See issue #1505.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1526)

### CODE REVIEW

**Comment by joeliedtke** [comment]
This is available to preview at:

https://deploy-preview-1526--competent-brattain-de2d6d.netlify.com/docs/pipelines/sdk/python-based-visualizations/

File: content/docs/pipelines/sdk/python-based-visualizations.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ arguments.
 
 ### Predefined TFX visualization
 
-<video width="960" height="540" controls>
+<video width="100%" max-width="100%" height="auto" max-height="100%" controls>
```
Comment (kbhawkey) [inline content/docs/pipelines/sdk/python-based-visualizations.md]: These settings fit the screen cast on the page (laptop screen). I have not verified a phone sized screen.
There are most likely other ways of making this element responsive.

**Comment by joeliedtke** [comment]
Thanks for making these changes! This looks good to me on a desktop and a mobile phone.

/lgtm
/approve

**Pull request merged**